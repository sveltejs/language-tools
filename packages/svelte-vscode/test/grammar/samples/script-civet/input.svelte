<script lang="civet">
    export abc = 123
</script>
