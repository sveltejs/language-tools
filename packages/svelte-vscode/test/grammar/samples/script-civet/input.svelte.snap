><script lang="civet">
#^ source.svelte meta.script.svelte meta.tag.start.svelte punctuation.definition.tag.begin.svelte
# ^^^^^^ source.svelte meta.script.svelte meta.tag.start.svelte entity.name.tag.svelte
#       ^ source.svelte meta.script.svelte meta.lang.civet.svelte meta.tag.start.svelte
#        ^^^^ source.svelte meta.script.svelte meta.lang.civet.svelte meta.tag.start.svelte meta.attribute.lang.svelte entity.other.attribute-name.svelte
#            ^ source.svelte meta.script.svelte meta.lang.civet.svelte meta.tag.start.svelte meta.attribute.lang.svelte punctuation.separator.key-value.svelte
#             ^ source.svelte meta.script.svelte meta.lang.civet.svelte meta.tag.start.svelte meta.attribute.lang.svelte string.quoted.svelte punctuation.definition.string.begin.svelte
#              ^^^^^ source.svelte meta.script.svelte meta.lang.civet.svelte meta.tag.start.svelte meta.attribute.lang.svelte string.quoted.svelte
#                   ^ source.svelte meta.script.svelte meta.lang.civet.svelte meta.tag.start.svelte meta.attribute.lang.svelte string.quoted.svelte punctuation.definition.string.end.svelte
#                    ^ source.svelte meta.script.svelte meta.lang.civet.svelte meta.tag.start.svelte punctuation.definition.tag.end.svelte
>    export abc = 123
#^^^^^^^^^^^^^^^^^^^^^ source.svelte meta.script.svelte meta.lang.civet.svelte
></script>
#^^ source.svelte meta.script.svelte meta.tag.end.svelte punctuation.definition.tag.begin.svelte
#  ^^^^^^ source.svelte meta.script.svelte meta.tag.end.svelte entity.name.tag.svelte
#        ^ source.svelte meta.script.svelte meta.tag.end.svelte punctuation.definition.tag.end.svelte
>